app/src/views/private/components/sidebar-detail.vue

```diff
@@ -1,185 +1,149 @@
-import { toRefs } from 'vue';
+import VBadge from '@/components/v-badge.vue';
+import VIcon from '@/components/v-icon/v-icon.vue';
+import { AccordionContent, AccordionHeader, AccordionItem, AccordionTrigger } from 'reka-ui';
+import { onUnmounted, watch } from 'vue';
+import { useSidebarStore } from '../private-view/stores/sidebar';
 
 const props = defineProps<{
-	icon: string;
+	id: string;
 	title: string;
+	icon: string;
 	badge?: boolean | string | number;
-	close?: boolean;
 }>();
 
+const sidebarStore = useSidebarStore();
+
 const emit = defineEmits<{
 	toggle: [open: boolean];
 }>();
 
-const { active, toggle } = useGroupable({
-	value: props.title,
-	group: 'sidebar-detail',
-});
-
-const appStore = useAppStore();
-const { sidebarOpen } = toRefs(appStore);
+watch(
+	() => sidebarStore.activeAccordionItem,
+	(newActive, oldActive) => {
+		if (newActive === props.id && newActive !== oldActive) {
+			emit('toggle', true);
+		} else if (oldActive === props.id && newActive !== props.id) {
+			emit('toggle', false);
+		}
+	},
+);
 
-function onClick() {
-	emit('toggle', !active.value);
-	toggle();
-}
+onUnmounted(() => {
+	if (sidebarStore.activeAccordionItem === props.id) {
+		sidebarStore.activeAccordionItem = undefined;
+	}
+});
 </script>
 
 <template>
-	<div class="sidebar-detail" :class="{ open: sidebarOpen }">
-		<button v-if="close" v-show="sidebarOpen" class="close" @click="sidebarOpen = false">
-			<v-icon name="close" />
-		</button>
-		<button v-tooltip.left="!sidebarOpen && title" class="toggle" :class="{ open: active }" @click="onClick">
-			<div class="icon">
-				<v-badge :dot="badge === true" bordered :value="badge" :disabled="!badge">
-					<v-icon :name="icon" />
-				</v-badge>
-			</div>
-			<div v-show="sidebarOpen" class="title">
-				{{ title }}
-			</div>
-			<div v-if="!close" class="icon">
-				<v-icon class="expand-icon" :name="active ? 'expand_less' : 'expand_more'" />
-			</div>
-		</button>
-		<transition-expand class="scroll-container">
-			<div v-show="active">
-				<div class="content">
-					<slot />
-				</div>
+	<AccordionItem class="accordion-item" :value="id">
+		<AccordionHeader>
+			<AccordionTrigger
+				v-tooltip.left="sidebarStore.collapsed && title"
+				class="accordion-trigger"
+				:class="{ collapsed: sidebarStore.collapsed }"
+			>
+				<VBadge :dot="badge === true" bordered :value="badge" :disabled="!badge">
+					<VIcon class="accordion-trigger-icon" :name="icon" />
+				</VBadge>
+				<span class="accordion-trigger-title">{{ title }}</span>
+				<VIcon class="accordion-trigger-chevron" name="chevron_left" />
+			</AccordionTrigger>
+		</AccordionHeader>
+		<AccordionContent class="accordion-content">
+			<div class="accordion-content-container">
+				<slot />
 			</div>
-		</transition-expand>
-	</div>
+		</AccordionContent>
+	</AccordionItem>
 </template>
 
 <style lang="scss" scoped>
-.sidebar-detail {
-	--v-badge-offset-x: 3px;
+.accordion-item {
+	display: contents;
```

Comment:
Let’s re-add the label size adjustment here, or find a differnet way of setting the field label font-size (e.g., add a prop to v-form, or set it to the fields object).

```suggestion
	display: contents;

	:deep(.type-label) {
		margin-block-end: 4px;
		font-size: 1rem;
	}
```